examples/worker/vite.config.ts

```diff
@@ -2,12 +2,23 @@ import { cloudflare } from '@flarelabs-net/vite-plugin-cloudflare';
 import { defineConfig } from 'vite';
 
 export default defineConfig({
+	dev: {
+		optimizeDeps: {
+			// Note: we want vite to pre-bundle this workspace dependency
+			include: ['@cloudflare-dev-module-resolution/builtins/cjs'],
+		},
+	},
 	plugins: [
 		cloudflare({
 			workers: {
 				worker: {
 					main: './src/index.ts',
 					wranglerConfig: './src/wrangler.toml',
+					overrides: {
+						// Note: this sets esbuild's platform to node, allowing it to accept
+						//       imports such as module, node:module, buffer, node:buffer, etc...
+						webCompatible: false,
+					},
```

Comment:
TODO: if a worker's config includes `nodejs_compat` then we want to set `webCompatible` to false in our plugin... I think?

Reply:
Done in: https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/32/commits/1d2ea14f10265041c324b93aa071979cb07c139c